### PR TITLE
New dog and profile scss

### PR DESCRIPTION
**Pull request by AaronD87**
New and profile scss and basic bootstrap containers

### CODE REVIEW

**Comment by ErlendLygre** [comment]
Looks horrible, but i'll accept it

**Comment by anthonyzamarro** [comment]
FREEEDOM

File: app/views/dogs/new.html.erb [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-<% end %>
+<div class="container">
+  <div class="row_new_dogs">
+    <div class="col-xs-12-col-sm-6-col-md-offset-4">
+      <h2 class="new_dog_header"> Add your dog </h2>
+      <%= simple_form_for @dog do |f| %>
+      <%= f.error_notification %>
+      <%= f.input :name %>
+      <%= f.input :species, label: "Breed" %>
+      <%= f.button :submit %>
```
Comment (anthonyzamarro) [inline app/views/dogs/new.html.erb:9]: I love this

File: app/assets/stylesheets/pages/_new_dog.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.new_dog_header {
+
+  text-align:center;
```
Comment (anthonyzamarro) [inline app/assets/stylesheets/pages/_new_dog.scss:3]: Gorgeous

**Comment by ErlendLygre** [comment]
CA CAAAW

**Comment by AaronD87** [comment]
Woo freedom!

**Pull request merged**